lib/config/options/index.spec.ts

```diff
@@ -38,7 +38,7 @@ describe('config/options/index', () => {
       .filter((option) => option.supportedPlatforms)
       .forEach((option) => {
         expect(option.supportedPlatforms).toBeNonEmptyArray();
-        for (const item of option.supportedPlatforms) {
+        for (const item of option.supportedPlatforms as string[]) {
```

Comment:
nit: wrap this in `is.nonEmptyArray` or `        assert.nonEmptyArray(option.supportedManagers)`

Reply:
can you make a suggestion please, didn't get it working nicely